### ISSUE TITLE
Why is the package stored in `python` instead of `imolecule`?

### ISSUE THREAD

**Issue by ctk3b** [patrickfuller/imolecule#13]
Hey Pat, I'm trying to use imolecule in another project and would need to make some small tweaks. I'll file a PR for them later but was curious about the choice to name the folder containing the package `python` instead of the more standard `imolecule`?

This prevents me from running `python setup.py develop` which is how I normally install packages I want to fiddle with. How do you do this locally? 

For the record, just changing the name of the folder and adjusting the entry in `setup.py` fixes this issue for me but I may just not be using it the same way that you are.

This is the relatively obvious output in case you're curious:
```python
$ python setup.py develop
Traceback (most recent call last):
  File "setup.py", line 2, in <module>
    from python import __version__
  File "/Users/ctk3b/imolecule/python/__init__.py", line 1, in <module>
    from imolecule.notebook import draw, generate, to_json
ImportError: No module named 'imolecule'
```

**Comment by ctk3b** [comment]
#14 shows the difference in case you're curious.

**Comment by patrickfuller** [comment]
Hey Christoph,

That's great!

Regarding the weird naming scheme, it's mainly because imolecule started as a javascript project. The IPython functionality was an add-on, but I think it's gradually become the main use of the project.

I'm open to re-organizing the project structure, but I think it needs a couple more changes to make it PyPI + node compatible. I'll leave some comments on the pull request.

**Comment by ctk3b** [comment]
Resolved in #14
